scripts/split-e2e-tests-codebuild.ts

```diff
@@ -100,7 +100,7 @@ export function loadConfigBase() {
   return yaml.load(fs.readFileSync(CODEBUILD_CONFIG_BASE_PATH, 'utf8'));
 }
 export function saveConfig(config: any): void {
-  const output = ['# auto generated file. DO NOT EDIT manually', yaml.dump(config, { noRefs: true })];
+  const output = ['# auto generated file. DO NOT EDIT manually', yaml.dump(config, { noRefs: true, lineWidth: -1 })];
```

Comment:
what does this do?

Reply:
This removes the minimum line width, which reduces the number of unnecessary newline characters.